src/ascii/port/callbacks.rs

```diff
@@ -0,0 +1,121 @@
+//! Callbacks for ports.
+#[cfg(doc)]
+use super::Port;
+use super::{Alert, Direction};
+
+/// A callback that is called after a packet is either transmitted or received.
+///
+/// See [`Port::set_packet_handler`] for more details.
+pub type PacketCallback<'a> = Box<dyn FnMut(&[u8], Direction) + 'a>;
```

Comment:
Naming is odd. The port methods call these "handlers" but the type alias is `Callback`. Probably better to standardize on `Handler`.

Reply:
That would technically be a breaking change, so should be done for v0.4.0 not v0.3.5.

Reply:
I think the best option is to
* deprecate these type aliases
* replaces them with `PacketHandler` and `SendPacketHandler`
* Rename the `LocalCallbacks` and `SendCallbacks` to `LocalHandlers` and `SendHandlers`
* Rename the trait `Callbacks` to `Handlers`, along with its associated types.

Reply:
Have replaced all uses of `Callback` with `Handler` (minus aliases). Using `Send` prefix.